src/test/java/seedu/address/logic/commands/ViewCommandTest.java

```diff
@@ -53,37 +49,40 @@ void execute_validIndex_success() throws CommandException {
 
     @Test
     void execute_invalidIndex_throwsCommandException() {
-        // Setup
         Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
         ViewCommand viewCommand = new ViewCommand(Index.fromZeroBased(10)); // Assuming there are fewer than 10 persons
 
-        // Execution and Assertions
         assertThrows(CommandException.class, () -> viewCommand.execute(model));
     }
 
     @Test
     void equals_sameObject_true() {
         ViewCommand viewCommand = new ViewCommand(Index.fromZeroBased(0));
 
-        // Assertions
-        assertTrue(viewCommand.equals(viewCommand));
+        assertEquals(viewCommand, viewCommand);
     }
 
     @Test
     void equals_differentObject_false() {
         ViewCommand viewCommand1 = new ViewCommand(Index.fromZeroBased(0));
         ViewCommand viewCommand2 = new ViewCommand(Index.fromZeroBased(1));
 
-        // Assertions
-        assertFalse(viewCommand1.equals(viewCommand2));
+        assertNotEquals(viewCommand1, viewCommand2);
     }
 
     @Test
     void equals_differentClass_false() {
         ViewCommand viewCommand = new ViewCommand(Index.fromZeroBased(0));
         Object otherObject = new Object();
 
-        // Assertions
-        assertFalse(viewCommand.equals(otherObject));
+        assertNotEquals(viewCommand, otherObject);
+    }
+
+    @Test
+    public void toString_validIndex_success() {
+        Index index = Index.fromZeroBased(3);
+        ViewCommand viewCommand = new ViewCommand(index);
+        String expectedToString = "seedu.address.logic.commands.ViewCommand{targetIndex=seedu.address.commons.core.index.Index{zeroBasedIndex=3}}";
+        assertEquals(expectedToString, viewCommand.toString());
     }
 }
```

Comment:
Great use of test cases !
- Good coverage of valid, invalid index
- Checking of equality of view commands
Great code overall, LGTM 👍🏾